server/sonar-web/src/main/js/apps/settings/components/PageHeader.js

```diff
@@ -35,7 +36,9 @@ export default class PageHeader extends React.PureComponent {
 
     const description =
       this.props.component != null
-        ? translate('project_settings.page.description')
+        ? this.props.branch
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3358)